apps/auth/views.py

```diff
@@ -22,6 +22,10 @@ def get_context(self) -> dict:
         }
 
     def get(self):
+        # if user is authorized he should not be able to login
+        if current_user.is_authenticated:
+            return redirect(url_for('auth.my_account'), code=302)
+
         return render_template(self.template_name, **self.get_context())
 
     def post(self):
```

Comment:
post method should also be protected, it is not possible to post from browser, but it may be done using external software for eg: Postman